packages/flutter/lib/src/material/about.dart

```diff
@@ -472,6 +482,314 @@ class LicensePage extends StatefulWidget {
 }
 
 class _LicensePageState extends State<LicensePage> {
+  final ValueNotifier<int> selectedId = ValueNotifier<int>(null);
+
+  @override
+  Widget build(BuildContext context) {
+    return _MasterDetailFlow(
+      detailPageFABlessGutterWidth: _getGutterSize(context),
+      title: Text(MaterialLocalizations.of(context).licensesPageTitle),
+      detailPageBuilder: _packageLicensePage,
+      masterViewBuilder: _packagesView,
+    );
+  }
+
+  Widget _packageLicensePage(BuildContext _, Object args, ScrollController sc) {
+    assert(args is _DetailArguments);
+    final _DetailArguments a = args as _DetailArguments;
+    return _PackageLicensePage(
+      packageName: a.packageName,
+      licenseEntries: a.licenseEntries,
+      scrollController: sc,
+    );
+  }
+
+  Widget _packagesView(final BuildContext _, final bool isLateral) {
+    final Widget about = _AboutProgram(
+        name: widget.applicationName ?? _defaultApplicationName(context),
+        icon: widget.applicationIcon ?? _defaultApplicationIcon(context),
+        version: widget.applicationVersion ?? _defaultApplicationVersion(context),
+        legalese: widget.applicationLegalese,
+      );
+    return _PackagesView(
+      about: about,
+      isLateral: isLateral,
+      selectedId: selectedId,
+    );
+  }
+}
+
+class _AboutProgram extends StatelessWidget {
+  const _AboutProgram({
+    Key key,
+    @required this.name,
+    @required this.version,
+    this.icon,
+    this.legalese,
+  })  : assert(name != null),
+        assert(version != null),
+        super(key: key);
+
+  final String name;
+  final String version;
+  final Widget icon;
+  final String legalese;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: EdgeInsets.symmetric(
+        horizontal: _getGutterSize(context),
+        vertical: 24.0,
+      ),
+      child: Column(
+        children: <Widget>[
+          Text(
+            name,
+            style: Theme.of(context).textTheme.headline5,
+            textAlign: TextAlign.center,
+          ),
+          if (icon != null)
+            IconTheme(data: Theme.of(context).iconTheme, child: icon),
+          Text(
+            version,
+            style: Theme.of(context).textTheme.bodyText2,
+            textAlign: TextAlign.center,
+          ),
+          Container(height: 18.0),
```

Comment:
Here, and the next `Container`, please define this constant as a separate static const variable to make it self documenting.

Also, consider using a `SizedBox` instead of a `Container`: `SizedBox` can be a const object, and so it's more performant.

Reply:
Thanks.